models/noise_generator.h

```diff
@@ -69,18 +72,19 @@ frequency double - Frequency of sine modulation in Hz
 Remarks:
  - All targets receive different currents.
  - The currents for all targets change at the same points in time.
+ - The interval between changes, dt, must be a multiple of the time step.
  - The effect of this noise current on a neuron DEPENDS ON DT. Consider
    the membrane potential fluctuations evoked when a noise current is
    injected into a neuron. The standard deviation of these fluctuations
    across an ensemble will increase with dt for a given value of std.
-   For the leaky integrate-and-fire neuron with time constant tau_m,
-   membrane potential fluctuations at times t_j+delay are given by
+   For the leaky integrate-and-fire neuron with time constant tau_m and 
```

Comment:
TravicCI code formatting complains here: there is whitespace at the end of the line. Please re-run clang-format or remove the whitespace.